extra_playbooks/build-cephfs-provisioner.yml

```diff
@@ -36,18 +36,18 @@
           git:
             repo: https://github.com/kubernetes-incubator/external-storage.git
             dest: "~/go/src/github.com/kubernetes-incubator"
-            version: 92295a30
+            version: a71a49d4
             clone: no
             update: yes
             
         - name: CephFS Provisioner | Build image
           shell: |
             cd ~/go/src/github.com/kubernetes-incubator/external-storage
-            REGISTRY=quay.io/kubespray/ VERSION=92295a30 make ceph/cephfs
+            REGISTRY=quay.io/kubespray/ VERSION=a71a49d4 make ceph/cephfs
 
         - name: CephFS Provisioner | Push image
           docker_image:
-            name: quay.io/kubespray/cephfs-provisioner:92295a30
```

Comment:
should this use the `cephfs_provisioner_image_repo` ?

Reply:
This is the tricky part of the helper playbook: because I will run it directly with localhost, e.g.
```
ansible-playbook -i localhost, extra_playbooks/build-cephfs-provisioner.yml 
```
After that confirm if the image build looks good enough, then manually push to quay.io.

Therefore we will never trigger the `roles/download`, and so not able to substitute with `cephfs_provisioner_image_repo`.

Reply:
@woopstar any update for this required, or shall we call ci to test this?